packages/utils/src/roles.js

```diff
@@ -28,12 +28,14 @@ export const isEnterpriseRole = role => ENTERPRISE_PERMISSIONS.some(permission =
  * @returns {boolean} true if the user has an enterprise role and/or the specified role, otherwise false
  */
 export const isEnterpriseUser = (user, role) => {
```

Comment:
I'd love it if there were tests for this.

Reply:
Tests have been added for the functions in roles.js 👍